visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.html

```diff
@@ -1,121 +1,138 @@
-				title="High Risk ({{ $ctrl._viewModel.riskProfile.highRisk }}% of code)"
-				class="high-risk"
-				style="width:{{ $ctrl._viewModel.riskProfile.highRisk }}%"
-				ng-disabled="true"
-			></md-button>
-			<md-button
-				ng-if="$ctrl._viewModel.riskProfile.veryHighRisk > 0"
-				title="Very High Risk ({{ $ctrl._viewModel.riskProfile.veryHighRisk }}% of code)"
-				class="very-high-risk"
-				style="width:{{ $ctrl._viewModel.riskProfile.veryHighRisk }}%"
-				ng-disabled="true"
-			></md-button>
-		</md-menu-item>
+			<md-menu-item class="ai-list-item" ng-show="$ctrl._viewModel.riskProfile === undefined">
+				<span>Could not calculate Risk Profile. Metrics rloc and mcc not available.</span>
+			</md-menu-item>
 
-		<md-menu-item class="ai-item" ng-show="$ctrl._viewModel.suspiciousMetricSuggestionLinks.length"
-			><md-button ng-disabled="true">
-				<span ng-show="$ctrl._viewModel.analyzedProgrammingLanguage.length"
-					>Suspicious Metrics in {{ $ctrl._viewModel.analyzedProgrammingLanguage }} code (click below to view)</span
-				>
-				<span ng-show="!$ctrl._viewModel.analyzedProgrammingLanguage.length">Suspicious Metrics (click below to view)</span>
-			</md-button></md-menu-item
-		>
+			<md-menu-item class="ai-list-item" ng-show="$ctrl._viewModel.riskProfile !== undefined">
+				<ul>
+					<li>{{ $ctrl._viewModel.riskProfile.lowRisk }}% of overall code is in files with low complexity</li>
+					<li>{{ $ctrl._viewModel.riskProfile.moderateRisk }}% of overall code is in files with moderate complexity</li>
+					<li>{{ $ctrl._viewModel.riskProfile.highRisk }}% of overall code is in files with high complexity</li>
+					<li>{{ $ctrl._viewModel.riskProfile.veryHighRisk }}% of overall code is in files with extreme complexity</li>
+				</ul>
+			</md-menu-item>
 
-		<md-menu-item class="custom-configs-item" ng-repeat="item in $ctrl._viewModel.suspiciousMetricSuggestionLinks">
-			<md-button
-				class="button-hovering"
-				ng-click="$ctrl.applyCustomConfig(item.generalCustomConfigId)"
-				title="Click to apply Map Configuration to show files with suspicious metric {{ item.metric }} and high risk"
-			>
-				Suspicious {{ item.metric.toUpperCase() }} Files
-			</md-button>
-			<md-button
-				class="button-hovering"
-				ng-if="item.outlierCustomConfigId.length > 0"
-				ng-click="$ctrl.applyCustomConfig(item.outlierCustomConfigId)"
-				title="Click to apply Map Configuration to show files with very high risk and metric values of {{ item.metric }}"
-			>
-				Very High Risk {{ item.metric.toUpperCase() }} Files
-			</md-button>
-		</md-menu-item>
-	</md-menu-content>
-</md-menu>
+			<md-menu-item class="ai-item risk-profile" ng-show="$ctrl._viewModel.riskProfile !== undefined">
+				<md-button
+					ng-if="$ctrl._viewModel.riskProfile.lowRisk > 0"
+					title="Low Risk ({{ $ctrl._viewModel.riskProfile.lowRisk }}% of code)"
+					class="low-risk"
+					style="width:{{ $ctrl._viewModel.riskProfile.lowRisk }}%"
```

Comment:
The width isn't floored. There are occurences where the sum of all widths is higher than 100%